app/components/autocomplete/autocomplete.tsx

```diff
@@ -124,10 +126,11 @@ const Autocomplete = ({
             s.push(style.shadow);
         }
         if (isSearch) {
-            s.push(style.base, style.searchContainer, {height: maxListHeight});
+            s.push(style.base, paddingTop ? {top: paddingTop} : style.searchContainer, {height: hasElements ? maxListHeight : 0});
```

Comment:
hasElements is needed to effectively remove the autocomplete screen until it has elements. Leaving it without elemements would allow showing the modifiers underneath, but you could not click on them.

Reply:
If we choose to use padding top, we can remove `style.searchContainer`

Reply:
I wonder if this was wrong to start with, and instead of `{height: maxListHeight}` should have been `{maxHeight: maxListHeight}`.

If so, you probably don't need to check `hasElements`.